html-forms/index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>HTML Forms</title>
+</head>
+<body>
+  <h1>Coding Form!</h1>
+  <form action="" method="post">
+    <div>
+      <label for="email">Email
+        <br>
+        <input type="text" name="email" id="email">
+      </label>
+      <br>
+      <label for="coding-reasons">Why is coding cool?
+        <br>
+        <select name="coding-reasons" id="coding-reasons">
+          <option value="select">Select an option</option>
+          <option value="interesting">Because it's interesting</option>
+          <option value="fun">Because it's fun</option>
+          <option value="powerful">Because it's powerful</option>
+        </select>
+      </label>
+    </div>
+    <br>
+    <div>
+      Favorite Language (choose one)
+    </div>
```

Comment:
This should be a `p` element. From the exercise instructions:

![image](https://user-images.githubusercontent.com/32880187/158477221-9ba809ea-d226-4aac-986d-b561b7cb4b87.png)